atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleOverflowWriteTable.java

```diff
@@ -176,9 +179,13 @@ public void putSentinels(Iterable<Cell> cells) {
     }
 
     @Override
-    public void update(Cell row, long ts, byte[] oldValue, byte[] newValue) {
-        throw new UnsupportedOperationException(
-                "Update is not supported for Oracle tables with table style 'overflow'.");
+    public void update(Cell cell, long ts, byte[] oldValue, byte[] newValue) {
+        if (oldValue.length > OVERFLOW_THRESHOLD || newValue.length > OVERFLOW_THRESHOLD) {
```

Comment:
nit: Since all checks are ` <= OVERFLOW_THRESHOLD`, will prefer if we could reverse the if. Will not push on this however.